src/gitops/components/application/ApplicationResourcesTab.tsx

```
@@ -2,42 +2,21 @@ import * as React from 'react';
-  FlexItem,
-  PageBody,
-  PageSection,
-  PageSectionVariants,
-  Title,
-} from '@patternfly/react-core';
-import { DataViewState } from '@patternfly/react-data-view/dist/esm/DataView';
-import { DataViewTh, DataViewTr } from '@patternfly/react-data-view/dist/esm/DataViewTable';
-import { CubesIcon } from '@patternfly/react-icons';
-import { Tbody, Td, Tr } from '@patternfly/react-table';
+import { useK8sModel, useUserSettings } from '@openshift-console/dynamic-plugin-sdk';
+import { Flex, FlexItem, PageBody, PageSection, PageSectionVariants, Title } from '@patternfly/react-core';
```

ESLint

[error] Replace `·Flex,·FlexItem,·PageBody,·PageSection,·PageSectionVariants,·Title·` with `⏎··Flex,⏎··FlexItem,⏎··PageBody,⏎··PageSection,⏎··PageSectionVariants,⏎··Title,⏎`

(prettier/prettier)